docs/posts/into-snowflake/index.qmd

```diff
@@ -0,0 +1,143 @@
+---
+title: "Snow IO: loading data from other DBs into Snowflake"
+author: "Phillip Cloud"
+error: false
+date: "2024-02-23"
+categories:
+  - blog
+  - snowflake
+  - io
+  - productivity
+---
+
+## Recap
+
+We've [blogged about Snowflake IO before](../snowflake-io/index.qmd), in the
+context of getting local files into Snowflake as fast as possible.
+
+In this post, we'll show how to insert query results from another system into
+Snowflake, using Ibis.
+
+## Setup
+
+### Connect to your non-Snowflake system
+
+We'll connect to a postgres database running locally in a container. You
+should be able to swap in your own connection details as needed.
```

Comment:
would it be worth mentioning `just up postgres` if they want to try this themselves? of course in practice they'll have their own database somewhere